plugins/modules/network_settings_intent.py

```diff
@@ -140,7 +156,7 @@
           ipv6_subnet:
             description: IPv6 Subnet address, example 2001 db8 85a3 0 100.
             type: str
-          ipv6TotalHost:
+          ipv6_total_host:
             description: IPv6 total host is required when ipv6_prefix value is false.
```

Comment:
The total number of hosts for IPv6 is required if the 'ipv6_prefix' option is set to false.